### PR TITLE
Firestore Console Build

### PR DESCRIPTION
**Pull request by Feiyang1**
Reattempt at creating the build target for console.
The original PR: https://github.com/firebase/firebase-js-sdk/pull/374

### CODE REVIEW

File: packages/firestore/tools/console.build.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+`;
+
+async function build() {
+  // create a bundle
+  const bundle = await rollup.rollup(inputOptions);
+
+  // generate code
+  const { code } = await bundle.generate(outputOptions);
+
+  const output = `${PREFIX}${JSON.stringify(String(code))}${POSTFIX}`;
```
Comment (tstirrat) [inline packages/firestore/tools/console.build.js]:
```js
JSON.stringify(String('a'));

// "\"a\""
```

I think that will output the code string wrapped.

```js
__firestore_exports__.firestore = "\"function(iife){}\"";
```

instead we want raw code:

```js
__firestore_exports__.firestore = function(iife){};
```

Comment (Feiyang1) [reply packages/firestore/tools/console.build.js]: They are wrapped in eval, so the extra quotes are needed to concat statements as strings.

File: packages/firestore/tools/console.build.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2018 Google Inc.
```
Comment (mikelehen) [inline packages/firestore/tools/console.build.js]: 2019

File: packages/firestore/tools/console.build.js (hunk 1 above)
Comment (tstirrat) [reply packages/firestore/tools/console.build.js]: An update on this.. found a simpler wrap:

```js
goog.module('firestore');
var firestore = function(e){}();  // note, not stringified, just pure rollup code
exports = firestore;
```

File: packages/firestore/index.console.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+export {
+  Firestore,
+  FirestoreDatabase,
+  PublicCollectionReference as CollectionReference,
+  PublicDocumentReference as DocumentReference,
+  PublicDocumentSnapshot as DocumentSnapshot,
+  PublicQuerySnapshot as QuerySnapshot
+} from './src/api/database';
+export { GeoPoint } from './src/api/geo_point';
+export { PublicBlob as Blob } from './src/api/blob';
```
Comment (tstirrat) [inline packages/firestore/index.console.ts]: We'll also need Timestamp exported

Comment (Feiyang1) [reply packages/firestore/index.console.ts]: Done.

**Pull request merged**